packages/react-composites/src/composites/CallComposite/Strings.tsx

```diff
@@ -213,6 +213,11 @@ export interface CallCompositeStrings {
    */
   peopleButtonLabel: string;
   /* @conditional-compile-remove(one-to-n-calling) @conditional-compile-remove(PSTN-calls) */
+  /**
+   * control bar People button label when checked
+   */
+  selectedPeopleButtonLabel: string;
```

Comment:
Why is this CC'd under these two?

Reply:
because peopleButtonLabel is cc under those two